tests/test_datetime_parse.py

```diff
@@ -175,6 +175,8 @@ def test_parse_python_format(delta):
         ('30', timedelta(seconds=30)),
         (30, timedelta(seconds=30)),
         (30.1, timedelta(seconds=30, milliseconds=100)),
+        (1e-5, timedelta(microseconds=10)),
```

Comment:
Above we have `'30'` and `30`. If I update this test to also have `'1e-5'` it fails as it doesn't handle parsing scientific notation as strings. 

https://github.com/samuelcolvin/pydantic/pull/3346 add support for scientific notation for strings. Would that work supplant this?

Reply:
Thanks for the review. Yes it seems like #3346 would handle these cases too. I'll close this then.